e2e/login.spec.ts

```diff
@@ -1,31 +1,49 @@
+import fs from 'fs';
+import path from 'path';
+import { fileURLToPath } from 'url';
 import { test, expect } from '@playwright/test';
-import projects from './projects-config.json' assert { type: 'json' };
 
-// Create a loop to loop over all the projects and create a test for everyone
+type Project = {
+  env: 'prod' | 'dev' | 'test';
+  name: string;
+  url: string;
+  title: string;
+  hasCookieBanner: boolean;
+};
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const data = fs.readFileSync(path.join(__dirname, 'projects-config.json')).toString();
 
-for (const project of Object.keys(projects)) {
-  const testProject = projects[project];
-  test(`As User, I can login to ${testProject.title}`, async ({ page }) => {
-    await page.goto(`${testProject.url}active/?app=${testProject.app}`);
+const appName = process.env.APP_NAME ?? 'all';
+const environment = process.env.ENVIRONMENT ?? 'prod';
+const projects: Project[] = JSON.parse(data)
+  .filter((project: Project) => project.env === environment)
+  .filter((project: Project) => appName === 'all' || project.name === appName);
```

Comment:
as it is about to be used for each test, better move it to a separate helper function